src/components/MainParticipantInfo/MainParticipantInfo.tsx

```diff
@@ -173,11 +208,18 @@ export default function MainParticipantInfo({ participant, children }: MainParti
           </Tooltip>
         )}
       </div>
-      {(!isVideoEnabled || isVideoSwitchedOff) && (
+      {!isVideoEnabled && (
         <div className={classes.avatarContainer}>
           <AvatarIcon />
         </div>
       )}
+      {isVideoSwitchedOff && (
+        <div className={clsx(classes.trackSwitchOffContainer, { [classes.isSwitchedOff]: isVideoSwitchedOff })}>
+          <Typography variant="body1" className={classes.switchedOffMessage}>
+            Video has been switched off to conserve bandwidth.
+          </Typography>
+        </div>
+      )}
```

Comment:
```suggestion
      <div className={clsx(classes.trackSwitchOffContainer, { [classes.isSwitchedOff]: isVideoSwitchedOff })}>
        <Typography variant="body1" className={classes.switchedOffMessage}>
          Video has been switched off to conserve bandwidth.
        </Typography>
      </div>
```
We don't need the `isVideoSwitchedOff &&` since the visibility is determined by the isSwitchedOff class. Plus, when we remove this element from the page when video is switched on, it prevents us from seeing the fade-out of the text.